app/main.py

```diff
@@ -1,3 +1,12 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    cache_dict = {}
+
+    def wrapper(*args, **kwargs):
+        cache_key = (args, frozenset(kwargs.items()))
+
+        if cache_key in cache_dict:
+            print("Getting from cache")
+            return cache_dict[cache_key]
```

Comment:
The else keyword was redundant, not the code inside else block. Make sure the flake8 and tests are working and send the task again